examples/semi_supervised_vae/vae_ssl_rws.py

```diff
@@ -117,11 +117,18 @@ def log_joint(observed):
                             [n_particles, 1, 1])
     proposal = labeled_proposal(x_labeled_ph, y_labeled_ph, n_z, n_particles)
     qz_samples, log_qz = proposal.query('z', outputs=True, local_log_prob=True)
-    labeled_cost, labeled_log_likelihood = zs.rws(
-        log_joint, {'x': x_labeled_obs, 'y': y_labeled_obs},
-        {'z': [qz_samples, log_qz]}, axis=0)
-    labeled_cost = tf.reduce_mean(labeled_cost)
-    labeled_log_likelihood = tf.reduce_mean(labeled_log_likelihood)
+
+    labeled_klpq_obj = zs.variational.klpq(log_joint,
+                                           observed={'x': x_labeled_obs,
+                                                     'y': y_labeled_obs},
+                                           latent={'z': [qz_samples, log_qz]},
+                                           axis=0)
+    labeled_variational_cost = tf.reduce_mean(labeled_klpq_obj.rws())
```

Comment:
Maybe two comments to show which is learning model parameters and which is adapting the proposal.